dev/src/reference.ts

```diff
@@ -2343,28 +2348,49 @@ export class Query<
    *
    * @private
    * @internal
-   * @param transactionIdOrReadTime A transaction ID or the read time at which
-   * to execute the query.
+   * @param transactionOrReadTime A transaction ID, options to start a new
+   *  transaction, or timestamp to use as read time.
    */
-  _get(
-    transactionIdOrReadTime?: Uint8Array | Timestamp
+  async _get(
+    transactionOrReadTime?: Uint8Array | Timestamp | api.ITransactionOptions
```

Comment:
Since the return type doesn't afford getting the new transactionId, I don't think we should allow passing in transaction options.

```suggestion
    transactionOrReadTime?: Uint8Array | Timestamp
```

Reply:
Done, btw.

Reply:
Done, btw.

Reply:
Done, btw.